logstash-core/lib/logstash/logging/logger.rb

```diff
@@ -6,6 +6,7 @@ class Logger
       java_import org.apache.logging.log4j.Level
       java_import org.apache.logging.log4j.LogManager
       java_import org.apache.logging.log4j.core.config.Configurator
+      @@config_mutex = Mutex.new
```

Comment:
Are you using a class var to make this kind of global? Could we do without it? (I am OK if the answer is that we need it)